iotdb-client/jdbc/src/test/java/org/apache/iotdb/jdbc/UtilsTest.java

```diff
@@ -65,6 +65,31 @@ public void testParseURL() throws IoTDBURLException {
     assertEquals(params.getPassword(), userPwd);
   }
 
+  @Test
+  public void testParseIPV6URL() throws IoTDBURLException {
+    String userName = "test";
+    String userPwd = "test";
+    String host1 =
+        "AD80::E32B::CR25::B3WE::DG4G::DWTF,AD80::E32B::CR25::B3WE::DG4G:DWTF::CGDE,AD80::E32B::CR25::B3WE::DG4G::DWTF";
```

Comment:
The IPv6 address permits only one "::", indicating the "0000" part. For instance, the AD80:E32B::CR25:B3WE:DG4G:DWTF means AD80:E32B:0000:0000:CR25:B3WE:DG4G:DWTF.

Reply:
Thanks ， I updated UT, please review again.